.github/workflows/ci.yaml

```diff
@@ -0,0 +1,35 @@
+name: ci
+
+on:
+  pull_request:
+    paths:
```

Comment:
Do we need `paths`?  Since there is just one package this feels like kinda overkill here.  We probably want it to run for everything.

Reply:
good call, removed!